frontend/src/core/utils/fluent/types.js

```diff
@@ -1,8 +1,19 @@
 /* @flow */
 
+export type FluentExpressionVariant = {
+    key: string,
+    value: FluentValue,
+};
+
+export type FluentExpression = {
+    type: string,
+    variants: Array<FluentExpressionVariant>,
+};
+
 export type FluentElement = {
     type: string,
     value: string,
+    expression: FluentExpression,
```

Comment:
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1580468 for the more general nomenclature problem, but I'd like to start here by not making it worse ;-)

We should have a `Placeable`, with a `SelectExpression` and just `Variant`s. And then `FluentElement` should probably be `{type,value} | Placeable` ?

Should `Variant` have `default` already, too? I don't think we use that yet, but it's probably nice to have for the next thing?